app/workers/janitor_worker.rb

```diff
@@ -7,7 +7,6 @@ def perform
     return unless ThreeScale.config.janitor_worker_enabled
 
     PurgeOldUserSessionsWorker.perform_async
-    PurgeStaleObjectsWorker.perform_later(EventStore::Event.name, DeletedObject.name)
-    true
```

Comment:
The `true` return value seems to be ignored by sidekiq. I don't know why it was put there. There is a remoed `assert` in a test further down related to this change.

 See:
https://github.com/sidekiq/sidekiq/commit/6ae5b0223600e80ec817c18d80982cc0877fa8bc#diff-6d086e98c7838c07b22b2db32d21260bf14a7a0575e957c4dc915e09a9b5831eR224